src/foam/core/NullServiceTest.java

```diff
@@ -0,0 +1,37 @@
+package foam.core;
```

Comment:
Copyright.

Reply:
Fixed in https://github.com/kgrgreer/foam3/pull/101/commits/d41fd31a0ae40be99c125b3e4345c3d67a4f7d0f.